### PR TITLE
Upgrade to symfony3.2.x #80

### PR DESCRIPTION
**Pull request by solivier**
need @skeleton's tests before to merge !

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,16 +12,16 @@
...
+        "symfony/symfony": "3.2.*",
         "doctrine/orm": "^2.4.8",
         "doctrine/doctrine-bundle": "~1.4",
         "symfony/swiftmailer-bundle": "~2.3",
         "symfony/monolog-bundle": "@stable",
         "sensio/distribution-bundle": "@stable",
         "sensio/framework-extra-bundle": "@stable",
         "incenteev/composer-parameter-handler": "@stable",
-        "hwi/oauth-bundle": "^0.4.0",
+        "hwi/oauth-bundle": "0.5.*-dev",
```
Comment (grena) [inline composer.json]: I remember having auth problem with the version above 0.4, but maybe I'm wrong.
Did you try to login with this version of oauth-bundle?

**Comment by skeleton** [comment]
I merged my pr about integration tests, you can rebase but be carefull they do not cover all the application

**Comment by skeleton** [comment]
You should check the new folders structure on sf3. Some files are moved in others folders, `var` folder is created

File: composer.json (hunk 1 above)
Comment (solivier) [inline composer.json]: No problems for me on my side I will do more tests to be sure

File: var/bootstrap.php.cache [hunk 2]
```diff
@@ -0,0 +1,1305 @@
+<?php
+
+ 
+namespace Symfony\Component\HttpFoundation
+{
```
Comment (skeleton) [inline var/bootstrap.php.cache]: Do not commit this file

**Pull request merged**